### PR TITLE
Update doc to use valid AMI and t2.micro instance types

### PR DESCRIPTION
**Pull request by catsby**
Small doc update to make sure we have valid AMI and instance type remains in the free tier

### CODE REVIEW

**Comment by phinze** [comment]
My grep shows a few more `m1.small`s floating around:

```
website/source/docs/configuration/resources.html.md:    instance_type = "m1.small"
website/source/docs/modules/usage.html.markdown:        instance_type = "m1.small"
website/source/docs/providers/aws/r/elasticache_cluster.html.markdown:    node_type = "cache.m1.small"
website/source/docs/providers/aws/r/instance.html.markdown:# Create a new instance of the ami-1234 on an m1.small node
website/source/docs/providers/aws/r/instance.html.markdown:    instance_type = "m1.small"
website/source/docs/providers/aws/r/launch_configuration.html.markdown:    instance_type = "m1.small"
website/source/docs/providers/aws/r/launch_configuration.html.markdown:    instance_type = "m1.small"
website/source/docs/providers/aws/r/launch_configuration.html.markdown:    instance_type = "m1.small"
website/source/index.html.erb:                                                          <p>    instance_type = <span class="txt-str">"m1.small"</span></p>
website/source/intro/examples/consul.html.markdown:and will default to "m1.small" if that key does not exist. Once the instance is created
```

**Comment by catsby** [comment]
@phinze cleaned up more. As discussed in chat, a few are actually valid

File: website/source/index.html.erb [hunk 1]
```diff
@@ -195,9 +195,9 @@
 								<p> </p>
 								<p>resource <span class="txt-spe">"aws_instance"</span> <span class="txt-str">"app"</span> {</p>
 								<p>    count = <span class="txt-int">5</span></p>
-								<p> </p>
-								<p>    ami = <span class="txt-str">"ami-043a5034"</span></p>
-								<p>    instance_type = <span class="txt-str">"m1.small"</span></p>
+                <p> </p>
```
Comment (phinze) [inline website/source/index.html.erb]: Whitespace attacks!

**Comment by phinze** [comment]
One whitespace nit, changes LGTM!

**Pull request merged**